src/pages/homepage.jsx

```diff
@@ -13,24 +13,42 @@ const Homepage = () => {
       loop: true,
       autoplay: true,
       animationData: waveAnimation, // Using animation JSON
-    });
+    })
 
-    return () => anim.destroy(); // Cleanup on unmount
-  }, []);
+    return () => anim.destroy() // Cleanup on unmount
+  }, [])
+
+  const handleButtonClick = () => {
+    window.gtag('event', 'button_click', {
+      event_category: 'button',
+      event_label: 'Discover More',
+      value: 1,
+    })
```

Comment:
currently we use     `ReactGA.send({ hitType: 'pageview', page: '/homepage' })`  I guess